packages/xo-server/src/xapi/index.mjs

```diff
@@ -1287,12 +1287,14 @@ export default class Xapi extends XapiBase {
     const host = this.pool.$master
     const sr = this.getObject(srId)
 
-    await this.call('host.call_plugin', host.$ref, 'xscontainer', 'create_config_drive', {
+    const vdiUuid = await this.call('host.call_plugin', host.$ref, 'xscontainer', 'create_config_drive', {
       vmuuid: vm.uuid,
       sruuid: sr.uuid,
       configuration: config,
     })
     await this.registerDockerContainer(vmId)
+
+    return vdiUuid
```

Comment:
The plugin returns the VDI UUID?

You were able to test both use case? (CoreOS and not CoreOS)

Reply:
- It returns the VDI UUID, I saw it in the xapi code

- I'm not able to test the case of creating a VM from CoreOS template (there is an error in the existing CoreOS template)